mixmasta/mixmasta.py

```diff
@@ -1018,6 +1019,31 @@ def normalizer(df: pd.DataFrame, mapper: dict, admin: str, gadm: gpd.GeoDataFram
                 else:
                     qualified_col_dict[k] = [kk]
 
+        # Convert aliases based on user annotations
+        aliases = feature_dict.get("aliases", {})
+        if aliases:
+            aliases_ = {}
+            # Need to handle cases where keys are ints/floats
+            # this could potentially break if the user stored numbers
+            # as strings since the converted int/float won't appropriately
+            # support the replace function
+            for kk, vv in aliases.items():
+                if kk.isnumeric():
```

Comment:
@mattprintz @GoogleSheets this is potentially of interest since it could bite us later.

When the user in Spacetag submits a `current` value of say `1` or `1.2` and a `new` value of `some string` we can't necessarily know or guarantee the type. 

For example, if the column they are trying to alias looked like:

| land_cover_type 	|
|-----------------	|
| 1               	|
| urban           	|
| savannah        	|
| urban           	|

And they are just trying to replace `1` with `grassland` we will have a problem because likely pandas will read the whole dataframe column as a `string`. So we will try to find/replace `1` with `grassland` when we should have tried to find/replace `"1"` (the `string`).